Update README

overcookedpanda opened the pull request:
Glad you liked them!  I've updated the README here.

#2

overcookedpanda committed:
Merge pull request #1 from ES-Collection/master

Merge pull request #2 from overcookedpanda/master

overcookedpanda committed:
Update README.md

overcookedpanda committed:
fix typo

GitBruno commented on README.md:
```diff
@@ -33,6 +39,8 @@ Simply clone this repo into the Photoshop scripts directory:
 
 The next time Photoshop is opened, you can find the scripts under `File -> Scripts`
 
+Note: 'Reg Marks and Labels.jsx' requires you to edit the location of `reg_mark.eps` for it to work properly, default location is on your desktop.
```
Can you remove this note and either change the path of `reg_mark.eps` to be relative to script or draw them in location. Here is a function to draw a line:

    function drawLine( startXY, endXY, width ) {
        // startXY : two element array of numbers for x,y start of line,
        // endXY   : two element array of numbers for x,y endof line,
        // width   : line width in pixels
        // color   : uses foreground color

        /* // SAMPLE USAGE

        var startPoint = [118,434];
        var endPoint = [335,434];
        var lineWidth = 2;
        drawLine( startPoint, endPoint, lineWidth );

        */

        var desc        = new ActionDescriptor();
        var lineDesc    = new ActionDescriptor();
        var startDesc   = new ActionDescriptor();
        var endDesc     = new ActionDescriptor();

        startDesc.putUnitDouble( charIDToTypeID('Hrzn'), charIDToTypeID('#Pxl'), startXY[0] );
        startDesc.putUnitDouble( charIDToTypeID('Vrtc'), charIDToTypeID('#Pxl'), startXY[1] );
        lineDesc.putObject(      charIDToTypeID('Strt'), charIDToTypeID('Pnt '), startDesc );
        endDesc.putUnitDouble(   charIDToTypeID('Hrzn'), charIDToTypeID('#Pxl'), endXY[0] );
        endDesc.putUnitDouble(   charIDToTypeID('Vrtc'), charIDToTypeID('#Pxl'), endXY[1] );
        lineDesc.putObject(      charIDToTypeID('End '), charIDToTypeID('Pnt '), endDesc );
        lineDesc.putUnitDouble(  charIDToTypeID('Wdth'), charIDToTypeID('#Pxl'), width );
        desc.putObject(          charIDToTypeID('Shp '), charIDToTypeID('Ln  '), lineDesc );
        desc.putBoolean(         charIDToTypeID('AntA'), true );
        executeAction(           charIDToTypeID('Draw'), desc, DialogModes.NO );
    };

overcookedpanda replied:
Changes made, many printers like to use custom registration marks, so I've left it as a separate file, but have made it relative to the Scripts directory.

GitBruno replied:
Cool, that looks good. I have updated relative path to use scriptPath instead, so we can fork this repo however we like. Thanks again for your contribution!

overcookedpanda committed:
change path for 'reg_mark.eps'

Path is now defaulted to Scripts directory.

overcookedpanda committed:
remove note

Commit:
Update Reg Marks and Labels.jsx

The path depends on how you fork the repo into your scripts folder. If we do `git clone Photoshop-Scripts myName` it breaks. Therefore it might be better to get the relative path from the script instead.

overcookedpanda reviewed (commented):
Very nice, I agree this is a better way to do it! Thanks for the update.
